nextflow.config

```diff
@@ -70,6 +70,13 @@ params {
     diversity_rarefaction_depth  = 500
     ancom_sample_min_count = 1
 
+    // Report options
+    report_template   = null
+    report_css        = null
+    report_logo       = null
```

Comment:
maybe it would be better to set the defaults here instead of in ampliseq.nf lines ~70+?

Reply:
Example from nf-core/rnaseq:
- [nextflow.config](https://github.com/nf-core/rnaseq/blob/3bec2331cac2b5ff88a1dc71a21fab6529b57a0f/nextflow.config#L55)
- [nextflow_schema.json](https://github.com/nf-core/rnaseq/blob/3bec2331cac2b5ff88a1dc71a21fab6529b57a0f/nextflow_schema.json#L150-L156)